DMTR-392.tex

```diff
@@ -40,8 +44,1009 @@
+
+Between w\_2022\_28 (2022-07-07; the source for pipelines version 24) and w\_2022\_48 (v25 source), we have made the following major changes.
+The pipelines now have the capability to generate HIPS trees from high-resolution, healpix-projected coadds with native Rubin code. This task was used to produce the HIPS images that are viewable in the Portal for Data Preview 0.2.
+\texttt{ScarletDeblendTask} now outputs a single storage container that allows users to access both the \texttt{scarlet} models and flux re-distributed models. This change uses an order of magnitude less storage space.
+A new package called \texttt{analysis\_tools} was added; \texttt{analysis\_tools} provides efficient, scalable, consistent and reproducible calculation of metrics and plots for quality assurance analyses. This will be the main resource for data-quality monitoring tools in the future.
+Also added in version 25 of the pipelines is the package \texttt{gbdes}, which will be used for astrometry in future versions of the science pipelines. The \texttt{gbdes} task will be available in version 26.
+Finally, v25 introduces an image subtraction refactor, where the combination of \texttt{subtractImages} and \texttt{detectAndMeasureDiaSources} has been introduced to replace the \texttt{imageDifference} task.
+
+All metrics reported here were calculated using the \href{https://github.com/lsst/faro}{faro} metric calculation package, which is part of the standard pipeline builds. All of the underlying algorithms to calculate metrics within \texttt{faro} are the same as they were in v24.1.0 of the Science Pipelines, so any differences between metrics from the v24 and v25 releases are due to changes in the underlying pipelines.
+
+The metric calculation pipelines from \texttt{faro} were run on the RC2 tracts to derive the photometric, astrometric, and shape metrics that are reported here. We exclude the two astrometry metrics (AM3 and AF3) that concern residuals on 200-arcminute scales, since the individual tracts of RC2 do not span large enough spatial scales to enable these measurements.
+
+For comparison, we provide the \SRD required ``design'' value of each Key Performance Metric (KPM) as defined in the Science Requirements Document \citedsp{LPM-17}. For the ellipticity correlation metrics, there are specifications only for \emph{r} and \emph{i} bands. The \emph{ugzy}-band measurements are of interest primarily for historical tracking.
```

Comment:
It might be interesting to add in the minimum spec for cases where we do not yet meet the design spec. For many metrics we will not achieve design goals until well into the survey.

Reply:
These have been updated on the figures.